Buildings/Experimental/DHC/Examples/Cooling/DistrictCooling.mo

```diff
@@ -0,0 +1,187 @@
+within Buildings.Experimental.DHC.Examples.Cooling;
```

Comment:
Hi @ChengnanShi-Work, you may have had different instruction from the GED team but one thing I wanted to verify. The original package structure we had for this example was `Buildings.Experimental.DHC.Examples.Cooling` (i.e., there isn't a separate `Cooling` package, but the `Cooling` model is in the `Examples` package directly.) Since there is only one model, a package isn't really necessary here. If you are planning to have multiple DC examples though, I'd suggest to change the `DistrictCooling` name so that the different types can be distinguished. 
FYI: We do not have a subpackage for the Steam example (Issue #2905), since there is only 1 model. Either way, we should coordinate to follow the same design. 

I defer to @JayHuLBL for direction, but I just wanted to comment.

Reply:
Thank you for the suggestions. I have changed the model's name to `Buildings.Experimental.DHC.Examples.Cooling.DirectUncontrolledETS`. Also, I added a schematic to illustrate the district cooling system. For the other two cooling system with `DirectControlled` and `Indirect` ETS, I plan to include them in the late future.